sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -222,7 +222,7 @@ _nib_onl_entry_t *_nib_onl_get(const ipv6_addr_t *addr, unsigned iface)
     for (unsigned i = 0; i < CONFIG_GNRC_IPV6_NIB_NUMOF; i++) {
         _nib_onl_entry_t *node = &_nodes[i];
 
-        if ((node->mode != _EMPTY) &&
+        if ((node->mode != _NC) &&
```

Comment:
Huh? The lines below check if the entry is a duplicate. Why only consider non-NCEs here instead of non-empty ones?

Reply:
oops, should have been `(node->mode & _NC)`